src/sentry/models/projecttemplate.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model, sane_repr
+
+
+@region_silo_only_model
+class ProjectTemplate(Model):
+    """
+    Identifies a project template that can be used to create new projects.
+
+    This model links the project template options to the organization that owns them.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    name = models.CharField(max_length=200)
+    organization = FlexibleForeignKey("sentry.Organization")
+    date_added = models.DateTimeField(default=timezone.now, null=True)
```

Comment:
You can subclass the `DefaultFieldsModel` ([here](https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/base.py#L299)) to get these